src/scripts/common.ts

```diff
@@ -34,3 +34,20 @@ export const StringsSeparatedByCommas: Type<string, string[]> = {
         return values;
     },
 };
+
+export const IDString: Type<string, string> = {
+    async from(str) {
+        if (_(str).isEmpty()) throw new Error("Value cannot be empty");
+        if (str.length !== 11) throw new Error("ID must be 11 char long");
+        return str;
+    },
+};
+
+export const IdsSeparatedByCommas: Type<string, string[]> = {
```

Comment:
Better to use the existing function`src/scripts/common.ts`. The string validation can be added there.

Reply:
You mean to use StringsSeparatedByCommas and add a handler in the userCase?
IdsSeparatedByCommas is a bit redundant, but that way we get a consistent options error messaging.
```
  datasets copy-org-units --url=http://admin:district@localhost:8080 --origin-dataset=LthvL9t2acW --destination-datasets=RvS8hay27O2,RvS8hay27O3,RvS8hay27O -r
                                                                                                  ^ IDs must be 11 char long
```